specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2021-03-01-preview/managedCassandra.json

```diff
@@ -0,0 +1,1102 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/clusterNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ClusterResource"
+            },
+            "description": "Parameters to provide for specifying the managed Cassandra cluster."
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The managed Cassandra cluster is being updated. Poll the provided operation for completion."
+          },
+          "200": {
+            "description": "Completed synchronously. This will only happen if the fields changed by patch require no changes to the actual Cassandra data center."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "cosmos-db.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/cassandraClusters/{clusterName}/repair": {
+      "post": {
+        "operationId": "CassandraClusters_RequestRepair",
+        "description": "Request that repair begin on this cluster as soon as possible.",
+        "x-ms-examples": {
+          "CosmosDBManagedCassandraRepair": {
+            "$ref": "./examples/CosmosDBManagedCassandraRepair.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/clusterNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/RepairPostBody"
+            },
+            "description": "Specification of what keyspaces and tables to run repair on."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success. The repair operation will begin as soon as possible."
+          },
+          "412": {
```

Comment:
So, we generally only want one response that is a failure, and that is the default response.  Any response code specifically modeled becomes a success in any generated clients.

Reply:
If this is a new api, we should likely remove this.  If not, we want to avoid the breaking change.

Reply:
Okay, going to remove it.